include/event_service_store.hpp

```diff
@@ -258,6 +258,31 @@ class EventServiceStore
     {
         return eventServiceConfig;
     }
+
+    bool needsWrite()
+    {
+        return needWrite;
+    }
+
+    void persistSubscription()
+    {
+        needWrite = true;
+        return;
+    }
+    void removeSubscription(const std::string& id)
+    {
+        auto obj = subscriptionsConfigMap.find(id);
+        if (obj != subscriptionsConfigMap.end())
+        {
+            BMCWEB_LOG_ERROR << "Deleting " << id
```

Comment:
Is this really needed as an LOG ERROR since you have 

                BMCWEB_LOG_ERROR << "TerminateAfterRetries is set. retryCount: "
                                 << retryCount << " .Subscriber: " << subId
                                 << " deleted";
                                 
                                 above?